### PR TITLE
[JENKINS-35020] Fixed some JSHint errors

### PR DESCRIPTION
**Pull request by recena**
[JENKINS-35020](https://issues.jenkins-ci.org/browse/JENKINS-35020)

These errors should be discussed:

```
[INFO] src/main/js/pluginSetupWizardGui.js: line 430, col 40, 'state' is defined but never used.
[INFO] src/main/js/pluginSetupWizardGui.js: line 954, col 16, 'setupWizardExtensions' is not defined.
[INFO] 
[INFO] 2 errors
[INFO] src/main/js/upgradeWizard.js: line 1, col 1, 'onSetupWizardInitialized' is not defined.
[INFO] 
[INFO] 1 error
[INFO] src/main/js/util/jenkins.js: line 201, col 49, 'receiver' is defined but never used.
[INFO] src/main/js/util/jenkins.js: line 200, col 32, 'Proxy' is not defined.
[INFO] 
[INFO] 2 errors
```

@reviewbybees esp @kzantow

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
It will definitely require backporting.
Could you please create a JIRA issue?

**Comment by recena** [comment]
@oleg-nenashev Jira issue filed and included in the PR description.

**Comment by kzantow** [comment]
@oleg-nenashev why should this be backported?

**Comment by oleg-nenashev** [comment]
@kzantow I mean backporting to 2.x.1 if we select a version without this fix

File: war/src/main/js/upgradeWizard.js [hunk 1]
```diff
@@ -1,40 +1,38 @@
-onSetupWizardInitialized(function(wizard) {
```
Comment (kzantow) [inline war/src/main/js/upgradeWizard.js]: :bug: this is necessary

Comment (recena) [inline war/src/main/js/upgradeWizard.js]: What do you propose?

Comment (kzantow) [inline war/src/main/js/upgradeWizard.js]: need to add a hint indicating this is a global method. it's something like `/*global onSetupWizardInitialized*/` see: http://stackoverflow.com/a/7866920

File: war/src/main/js/util/jenkins.js [hunk 2]
```diff
@@ -193,12 +193,13 @@ exports.loadTranslations = function(bundleName, handler, onError) {
 			}
 			throw 'Unable to load localization data: ' + res.message;
 		}
-		
+
 		var translations = res.data;
-		
-		if('undefined' !== typeof(Proxy)) {
+
+		/* globals Proxy: true */
```
Comment (kzantow) [inline war/src/main/js/util/jenkins.js]: ah yes, like this 👍

File: war/src/main/js/upgradeWizard.js (hunk 1 above)
Comment (recena) [inline war/src/main/js/upgradeWizard.js]: I did that in other cases, the question is: Why do you need `onSetupWizardInitialized`?

Comment (recena) [inline war/src/main/js/upgradeWizard.js]: @kzantow What do you think about [this](https://github.com/jenkinsci/jenkins/pull/2368/commits/f743ef9311523ca2ae0cd82823a1082cb8e62119?w=1)?

File: war/src/main/js/upgradeWizard.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-	var $ = wizard.$; // jQuery
+function onSetupWizardInitialized(wizard) {
```
Comment (kzantow) [inline war/src/main/js/upgradeWizard.js]: :bug: No, please revert this file and just add `/* global onSetupWizardInitialized */` to the first line

Comment (recena) [inline war/src/main/js/upgradeWizard.js]: @kzantow done but I disagree.

**Comment by kzantow** [comment]
:bee:

**Comment by recena** [comment]
@jenkinsci/code-reviewers

File: war/src/main/js/pluginSetupWizardGui.js [hunk 4]
```diff
@@ -427,7 +427,7 @@ var createPluginSetupWizard = function(appendTarget) {
 	};
 	
 	// Define actions
-	var showInstallProgress = function(state) {
+	var showInstallProgress = function() {
```
Comment (oleg-nenashev) [inline war/src/main/js/pluginSetupWizardGui.js]: This parameter is unused, right?

**Comment by oleg-nenashev** [comment]
LGTM. 🐝 if it works

File: war/src/main/js/pluginSetupWizardGui.js (hunk 4 above)
Comment (recena) [inline war/src/main/js/pluginSetupWizardGui.js]: @oleg-nenashev No, for that reason there was a JSHint error 😄

**Comment by recena** [comment]
@reviewbybees done

**Pull request merged**